arduino-ide-extension/src/browser/widgets/sketchbook/sketchbook-tree-widget.tsx

```diff
@@ -22,6 +26,11 @@ import { SelectableTreeNode } from '@theia/core/lib/browser/tree/tree-selection'
 import { Sketch } from '../../contributions/contribution';
 import { nls } from '@theia/core/lib/common';
 
+const customTreeProps: TreeProps = {
+  leftPadding: 20,
```

Comment:
> About `expansionTogglePadding: 6`, it has been set because doing some testing it seemed to be the correct value to use to fix this issue.

Thank you!

> What would you suggest?

Keep it as is.